This isn't building on my machine:

```
make: Entering directory '/home/ack/work/node-webrtc/build'
  CXX(target) Release/obj.target/webrtc/src/binding.o
  CXX(target) Release/obj.target/webrtc/src/peerconnection.o
In file included from ../src/peerconnection.h:19:0,
                 from ../src/peerconnection.cc:16:
../src/peerconnection.cc: In static member function ‘static void PeerConnection::Run(uv_async_t*, int)’:
../node_modules/nan/nan.h:394:40: error: conversion from ‘v8::Persistent<v8::Object>’ to non-scalar type ‘v8::Local<v8::Object>’ requested
 # define NanObjectWrapHandle(obj) obj->handle_
                                        ^
../src/peerconnection.cc:146:31: note: in expansion of macro ‘NanObjectWrapHandle’
   v8::Local<v8::Object> lpc = NanObjectWrapHandle(self);
                               ^
webrtc.target.mk:147: recipe for target 'Release/obj.target/webrtc/src/peerconnection.o' failed
make: *** [Release/obj.target/webrtc/src/peerconnection.o] Error 1
make: Leaving directory '/home/ack/work/node-webrtc/build'
```

Maybe I'm missing something?

--- a/AUTHORS
+++ b/AUTHORS
@@ -2,3 +2,4 @@ Alan K <ack@modeswitch.org> (blog.modeswitch.org)
 Eric Rescorla <ekr@rtfm.org>
 Pasquale Boemio <boemianrapsodi@gmail.com>
 Damon Oehlman <damon@fluxant.com>
+Benjamin Byholm <bbyholm@abo.fi>

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
   "dependencies": {
     "bindings": "~1.1.1",
     "node-gyp": "~0.10.10",
-    "nan": "~0.4.1"
+    "nan": "~0.4.4"
   },
   "scripts": {
     "preinstall": "make prepare_env build_deps",

--- a/src/datachannel.cc
+++ b/src/datachannel.cc
@@ -70,7 +70,7 @@ void DataChannel::Run(uv_async_t* handle, int status)
   TRACE_CALL;
   NanScope();
   DataChannel* self = static_cast<DataChannel*>(handle->data);
-  v8::Local<v8::Object> ldc = NanObjectWrapHandle(self);
+  v8::Local<v8::Object> dc = NanObjectWrapHandle(self);
 
   while(true)
   {
@@ -89,19 +89,19 @@ void DataChannel::Run(uv_async_t* handle, int status)
     if(DataChannel::ERROR & evt.type)
     {
       DataChannel::ErrorEvent* data = static_cast<DataChannel::ErrorEvent*>(evt.data);
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(ldc->Get(String::New("onerror")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(dc->Get(String::New("onerror")));
       v8::Local<v8::Value> argv[1];
       argv[0] = Exception::Error(String::New(data->msg.c_str()));
-      callback->Call(ldc, 1, argv);
+      callback->Call(dc, 1, argv);
     } else if(DataChannel::STATE & evt.type)
     {
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(ldc->Get(String::New("onstatechange")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(dc->Get(String::New("onstatechange")));
       v8::Local<v8::Value> argv[0];
-      callback->Call(ldc, 0, argv);
+      callback->Call(dc, 0, argv);
     } else if(DataChannel::MESSAGE & evt.type)
     {
       MessageEvent* data = static_cast<MessageEvent*>(evt.data);
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(ldc->Get(String::New("onmessage")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(dc->Get(String::New("onmessage")));
 
       Local<Object> array = NanPersistentToLocal(ArrayBufferConstructor)->NewInstance();
       array->SetIndexedPropertiesToExternalArrayData(data->message, v8::kExternalByteArray, data->size);
@@ -112,7 +112,7 @@ void DataChannel::Run(uv_async_t* handle, int status)
 
       v8::Local<v8::Value> argv[1];
       argv[0] = array;
-      callback->Call(ldc, 1, argv);
+      callback->Call(dc, 1, argv);
     }
     // FIXME: delete event
   }

--- a/src/datachannel.cc
+++ b/src/datachannel.cc
@@ -70,7 +70,7 @@ void DataChannel::Run(uv_async_t* handle, int status)
   TRACE_CALL;
   NanScope();
   DataChannel* self = static_cast<DataChannel*>(handle->data);
-  v8::Local<v8::Object> dc = NanObjectWrapHandle(self);
+  v8::Handle<v8::Object> dc = NanObjectWrapHandle(self);
 
   while(true)
   {

--- a/src/peerconnection.cc
+++ b/src/peerconnection.cc
@@ -143,7 +143,7 @@ void PeerConnection::Run(uv_async_t* handle, int status)
   NanScope();
 
   PeerConnection* self = static_cast<PeerConnection*>(handle->data);
-  v8::Local<v8::Object> lpc = NanObjectWrapHandle(self);
+  v8::Local<v8::Object> pc = NanObjectWrapHandle(self);
 
   while(true)
   {
@@ -162,63 +162,63 @@ void PeerConnection::Run(uv_async_t* handle, int status)
     if(PeerConnection::ERROR_EVENT & evt.type)
     {
       PeerConnection::ErrorEvent* data = static_cast<PeerConnection::ErrorEvent*>(evt.data);
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(lpc->Get(String::New("onerror")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(String::New("onerror")));
       v8::Local<v8::Value> argv[1];
       argv[0] = Exception::Error(String::New(data->msg.c_str()));
-      callback->Call(lpc, 1, argv);
+      callback->Call(pc, 1, argv);
     } else if(PeerConnection::SDP_EVENT & evt.type)
     {
       PeerConnection::SdpEvent* data = static_cast<PeerConnection::SdpEvent*>(evt.data);
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(lpc->Get(String::New("onsuccess")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(String::New("onsuccess")));
       v8::Local<v8::Value> argv[1];
       argv[0] = String::New(data->desc.c_str());
-      callback->Call(lpc, 1, argv);
+      callback->Call(pc, 1, argv);
     } else if(PeerConnection::VOID_EVENT & evt.type)
     {
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(lpc->Get(String::New("onsuccess")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(String::New("onsuccess")));
       v8::Local<v8::Value> argv[0];
-      callback->Call(lpc, 0, argv);
+      callback->Call(pc, 0, argv);
     } else if(PeerConnection::SIGNALING_STATE_CHANGE & evt.type)
     {
       PeerConnection::StateEvent* data = static_cast<PeerConnection::StateEvent*>(evt.data);
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(lpc->Get(String::New("onsignalingstatechange")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(String::New("onsignalingstatechange")));
       if(!callback.IsEmpty())
       {
         v8::Local<v8::Value> argv[1];
         argv[0] = Uint32::New(data->state);
-        callback->Call(lpc, 1, argv);
+        callback->Call(pc, 1, argv);
       }
     } else if(PeerConnection::ICE_CONNECTION_STATE_CHANGE & evt.type)
     {
       PeerConnection::StateEvent* data = static_cast<PeerConnection::StateEvent*>(evt.data);
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(lpc->Get(String::New("oniceconnectionstatechange")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(String::New("oniceconnectionstatechange")));
       if(!callback.IsEmpty())
       {
         v8::Local<v8::Value> argv[1];
         argv[0] = Uint32::New(data->state);
-        callback->Call(lpc, 1, argv);
+        callback->Call(pc, 1, argv);
       }
     } else if(PeerConnection::ICE_GATHERING_STATE_CHANGE & evt.type)
     {
       PeerConnection::StateEvent* data = static_cast<PeerConnection::StateEvent*>(evt.data);
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(lpc->Get(String::New("onicegatheringstatechange")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(String::New("onicegatheringstatechange")));
       if(!callback.IsEmpty())
       {
         v8::Local<v8::Value> argv[1];
         argv[0] = Uint32::New(data->state);
-        callback->Call(lpc, 1, argv);
+        callback->Call(pc, 1, argv);
       }
     } else if(PeerConnection::ICE_CANDIDATE & evt.type)
     {
       PeerConnection::IceEvent* data = static_cast<PeerConnection::IceEvent*>(evt.data);
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(lpc->Get(String::New("onicecandidate")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(String::New("onicecandidate")));
       if(!callback.IsEmpty())
       {
         v8::Local<v8::Value> argv[3];
         argv[0] = String::New(data->candidate.c_str());
         argv[1] = String::New(data->sdpMid.c_str());
         argv[2] = Integer::New(data->sdpMLineIndex);
-        callback->Call(lpc, 3, argv);
+        callback->Call(pc, 3, argv);
       }
     } else if(PeerConnection::NOTIFY_DATA_CHANNEL & evt.type)
     {
@@ -227,12 +227,12 @@ void PeerConnection::Run(uv_async_t* handle, int status)
       cargv[0] = v8::External::New(static_cast<void*>(dci));
       v8::Local<v8::Value> dc = NanPersistentToLocal(DataChannel::constructor)->NewInstance(1, cargv);
 
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(lpc->Get(String::New("ondatachannel")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(String::New("ondatachannel")));
       if(!callback.IsEmpty())
       {
         v8::Local<v8::Value> argv[1];
         argv[0] = dc;
-        callback->Call(lpc, 1, argv);
+        callback->Call(pc, 1, argv);
       }
     }
     // FIXME: delete event

--- a/src/peerconnection.cc
+++ b/src/peerconnection.cc
@@ -143,7 +143,7 @@ void PeerConnection::Run(uv_async_t* handle, int status)
   NanScope();
 
   PeerConnection* self = static_cast<PeerConnection*>(handle->data);
-  v8::Local<v8::Object> pc = NanObjectWrapHandle(self);
+  v8::Handle<v8::Object> pc = NanObjectWrapHandle(self);
 
   while(true)
   {

--- a/src/peerconnection.h
+++ b/src/peerconnection.h
@@ -74,7 +74,9 @@ class SetRemoteDescriptionObserver :
     virtual void OnFailure( const std::string& msg );
 };
 
-class PeerConnection : public ObjectWrap, public webrtc::PeerConnectionObserver
+class PeerConnection
+: public ObjectWrap,
+  public webrtc::PeerConnectionObserver
 {
 
 public:

